### PR TITLE
Fix VaInput range in slots example of slider docs

### LINKED ISSUE #3977: VaSlider: Limit circle draw range from 0 to 100
is it right?
slider

### PR DESCRIPTION
**Pull request by xiongmao86**
Set limit on the VaInput in order to limit sliding circle inside slider.
Fixes #3977

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/docs/page-config/ui-elements/slider/examples/Slots.vue [hunk 1]
```diff
@@ -11,7 +12,8 @@
   <va-slider v-model="value2">
     <template #append>
       <va-input
-        v-model.number="value2"
+        v-model="value2Proxy"
+        @input="e => e.target.value = value2Proxy"
```
Comment (m0ksem) [inline packages/docs/page-config/ui-elements/slider/examples/Slots.vue:16]: Maybe we should do it inside VaInput. I feel like it is not obvious to user (and for the one who wrote this doc example too). I'd expect v-model to be the same as input.value. What do you think?

**Review by m0ksem — APPROVED** [review]

File: packages/docs/page-config/ui-elements/slider/examples/Slots.vue (hunk 1 above)
Comment (xiongmao86) [reply packages/docs/page-config/ui-elements/slider/examples/Slots.vue:16]: Hi, @m0ksem, I agree with you, I will try to put it inside VaInput.

Comment (xiongmao86) [reply packages/docs/page-config/ui-elements/slider/examples/Slots.vue:16]: Hi, @m0ksem, I think I can put the logic in `onInput` function in the `useCleave` hook, but I am not sure that is a good idea, but I can't find a way to put into the VaInput.vue. Can you give me some pointers?

File: packages/ui/src/components/va-input/VaInput.vue [hunk 2]
```diff
@@ -164,6 +167,25 @@ export default defineComponent({
...
+        if (typeof min.value !== 'undefined' && value < min.value) {
+          valueComputed.value = min.value
+          return
+        } else if (typeof max.value !== 'undefined' && value > max.value) {
+          valueComputed.value = max.value
+          return
+        }
+      }
+      valueComputed.value = value
+    }
```
Comment (xiongmao86) [inline packages/ui/src/components/va-input/VaInput.vue:188]: Hi, @m0ksem, I don't know why this wouldn't work. I can't see this in vue devtool, doesn't show any prop or variable.

File: packages/docs/page-config/ui-elements/slider/examples/Slots.vue (hunk 1 above)
Comment (m0ksem) [reply packages/docs/page-config/ui-elements/slider/examples/Slots.vue:16]: I think you can manually add `@input` event to handle it separately from useCleave.

File: packages/ui/src/components/va-input/VaInput.vue (hunk 2 above)
Comment (m0ksem) [reply packages/ui/src/components/va-input/VaInput.vue:188]: I think it is better to use `VaCounter` instead of `VaInput type="number"`

**Review by m0ksem — APPROVED** [review]

**Pull request merged**